lib/src/cli/cli.dart

```diff
@@ -1,12 +1,11 @@
 import 'dart:async';
-
+import 'package:collection/collection.dart';
```

Comment:
This is lowering our pana score a bit. We should include this package in the pubspec.

Reply:
@renancaraujo fixed.